client/my-sites/comments/comment-list/index.jsx

```diff
@@ -3,11 +3,18 @@
  */
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { filter, map } from 'lodash';
+import { localize } from 'i18n-calypso';
+import { filter, keyBy, map } from 'lodash';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
```

Comment:
I think we're now OK with the React upgrade, but I recall this component leaving classes on elements in IE11. It'd be worth doing a quick test in that env.

https://github.com/Automattic/wp-calypso/issues/982